src/rpc/blockchain.cpp

```diff
@@ -2855,6 +2855,9 @@ return RPCHelpMan{
         return data;
     };
 
+    // We present the IBD/background chainstate as "normal" to users, since
+    // it should not matter to them if there is also a snapshot chainstate.
+    // To users, the IBD chainstate should always be considered "normal".
```

Comment:
21ad0c53760cf051b4daa177885db1358785d17a: this comment can be dropped now afaik since #28590 was merged.

Reply:
yepp, done